numpy/core/src/multiarray/_datetime.h

```diff
@@ -183,7 +183,7 @@ convert_datetime_metadata_tuple_to_datetime_metadata(PyObject *tuple,
  * the Python datetime.tzinfo object.
  */
 NPY_NO_EXPORT int
-get_tzoffset_from_pytzinfo(PyObject *timezone, npy_datetimestruct *dts);
+get_tzoffset_from_pytzinfo(PyObject *timezone_obj, npy_datetimestruct *dts);
```

Comment:
Was reserved word?

Reply:
Was not consistent with the name in the implementation, and changing this one was easier